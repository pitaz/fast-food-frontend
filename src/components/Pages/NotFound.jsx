import React from 'react';

const NotFound = () => (
  <div className='container'>
    <h3 className='notfound'>Page not Found 404</h3>
    
  </div>
);

export default NotFound;
