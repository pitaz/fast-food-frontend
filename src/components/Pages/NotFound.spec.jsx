/* eslint-disable react/no-children-prop */
/* eslint-disable no-undef */
import React from 'react';
import expect from 'expect';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NotFound from './NotFound.jsx';


let wrapper;

describe('The Signin Component Test Suite', () => {
  wrapper = shallow(<NotFound />);
  it('renders the menu component', () => {
    expect(wrapper.exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
  }); 

  it('should match the snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('should render <MenuForm /> Component', () => {
    expect(wrapper.length).toBe(1);
    expect(wrapper.find(NotFound)).toBeDefined();
  });
});
