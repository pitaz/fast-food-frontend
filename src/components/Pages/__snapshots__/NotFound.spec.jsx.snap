// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Signin Component Test Suite renders the menu component 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h3
        className="notfound"
      >
        Page not Found 404
      </h3>,
      "className": "container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Page not Found 404",
        "className": "notfound",
      },
      "ref": null,
      "rendered": "Page not Found 404",
      "type": "h3",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h3
          className="notfound"
        >
          Page not Found 404
        </h3>,
        "className": "container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Page not Found 404",
          "className": "notfound",
        },
        "ref": null,
        "rendered": "Page not Found 404",
        "type": "h3",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;

exports[`The Signin Component Test Suite should match the snapshot 1`] = `
<div
  className="container"
>
  <h3
    className="notfound"
  >
    Page not Found 404
  </h3>
</div>
`;
